### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by pornel**
Updated security docs and added a page dedicated to App Transport Security

### CODE REVIEW

File: documentation/apptransportsecurity/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (jakepetroules) [inline documentation/apptransportsecurity/index.md]: Please name this file `documentation/app-transport-security/index.md`

File: documentation/apptransportsecurity/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+id: documentation
+title: Sparkle App Transport Security
+---
+## App Transport Security
+
+Apple has deprecated insecure HTTP in OS X 10.11 (El Capitan). Applications compiled with 10.11 SDK are by default **blocked from using HTTP** and will not be able to download any updates over HTTP. You **must** either:
+
+* Switch to HTTPS entirely. Change the appcast URL in Info.plist and download URLs in the appcast to use HTTPS. This is recommended, and you can get free certificates from [Let's encrypt](https://letsencrypt.org/).
+
+* Add an exception to Info.plist in your app if you can't stop using HTTP yet. [The exception properties are in Apple's documentation](https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/).
```
Comment (jakepetroules) [inline documentation/apptransportsecurity/index.md]: Replace 'ios' with 'mac' here. While the page is the same... this is an OS X project. ;)

File: documentation/index.md [hunk 3]
```diff
@@ -29,57 +29,64 @@ If you are using [CocoaPods](https://cocoapods.org), then follow these [alternat
...
-  * Code-signing the application via Apple's Developer ID program or with your own certificate
-  * Specifying a [DSA signature](https://en.wikipedia.org/wiki/Digital_signature) of the SHA-1 hash of the published update archive and including a public DSA key inside your update
+Since Sparkle is downloading executable code to your users' systems, you must be very careful about security. To let Sparkle know that a downloaded update is not corrupted and came from you (instead of a malicious attacker), we recommend:
 
-* If you are not serving updates over https and you are linking against the 10.11 or later SDK, you must comply with Apple's [App Transport Security](https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/). In short, http requests will be rejected by default unless an exception is added within your app. App Transport Security has other specific requirements too, so please test your app on 10.11 or later even if you already are serving over https!
+  * Serving the update over HTTPS (your app *will not update on OS X 10.11* unless you comply with Apple's [App Transport Security](/documentation/apptransportsecurity/) requirements).
```
Comment (jakepetroules) [inline documentation/index.md]: change apptransportsecurity URL here too

File: documentation/index.md [hunk 4]
```diff
@@ -29,57 +29,64 @@ If you are using [CocoaPods](https://cocoapods.org), then follow these [alternat
...
+  * Code-signing the application via Apple's Developer ID program or,
+  * specifying a DSA [signature](https://en.wikipedia.org/wiki/Digital_signature) of the SHA-1 hash of the published update archive and including a public DSA key inside your update.
 
-* While signing your archive using DSA signatures strengthens your update and may open up to additional features, they are not required to be used if the update is:
-  * Served over https (serving over http without using DSA signatures is now deprecated)
-  * Packaged as a regular app (i.e, not a preference pane, plug-in, package installer, or some other type)
-  * Distributed without providing binary delta patches for more efficient updates
+If you are not serving updates over HTTPS and you are linking against the 10.11 SDK, **your updates will be blocked by OS X El Capitan**. You *must* comply with Apple's [App Transport Security](/documentation/apptransportsecurity/). In short, HTTP requests will be rejected by the system unless an exception is added within your app. App Transport Security has other specific requirements too, so please test updating your app on 10.11 even if you already are serving over HTTPS!
```
Comment (jakepetroules) [inline documentation/index.md]: change apptransportsecurity URL here too

File: documentation/index.md [hunk 5]
```diff
@@ -29,57 +29,64 @@ If you are using [CocoaPods](https://cocoapods.org), then follow these [alternat
...
 
-* Sparkle uses appcasts to get information about software updates.
-* An appcast is an RSS feed with some extra information for Sparkle's purposes.
+Sparkle uses appcasts to get information about software updates. An appcast is an RSS feed with some extra information for Sparkle's purposes.
+
 * Make a copy of the sample appcast included in the Sparkle distribution.
 * Read the sample appcast to familiarize yourself with the format, then edit out all the items and add one for the new version of your app by following the instructions at [Publishing an update](/documentation/publishing#publishing-an-update).
 * Upload your appcast to a webserver.
-* Add a <code>SUFeedURL</code> key to your Info.plist; set its value to the URL of your appcast. We strongly encourage you to use HTTPS URLs for the appcast.
+* Add a <code>SUFeedURL</code> key to your Info.plist; set its value to the URL of your appcast. We [strongly encourage you to use HTTPS](/documentation/apptransportsecurity/) URLs for the appcast.
```
Comment (jakepetroules) [inline documentation/index.md]: here too

File: documentation/apptransportsecurity/index.md (hunk 1 above)
Comment (pornel) [inline documentation/apptransportsecurity/index.md]: done

**Comment by jakepetroules** [comment]
If you rebase it should pass CI.

**Pull request merged**